_training/curriculum.md

```diff
@@ -21,23 +21,23 @@ The curriculum is comprised of a set of standardized *modules*, so that students
 
 ### C++ corner
 
-{% include list_of_selected_training_modules.html ids="hepcpp,moderncpp,cmake"%}
+{% include list_of_selected_training_modules.html ids="hepcpp,cmake"%}
```

Comment:
Should we add moderncpp to archived too then?

Reply:
👍  Ah sure. Though there is really not much to be seen there.